### ISSUE TITLE
Cannot install

### ISSUE THREAD

**Issue by mfzl** [zchee/deoplete-go#62]
Trying to install gives me the following error

```
error: no such remote ref 1356d8cd3593b1abb2f291e2cabf5430dfa108ed
Fetched in submodule path 'rplugin/python3/deoplete/clang', but it did not contain 1356d8cd3593b1abb2f291e2cabf5430dfa108ed. Direct fetching of that commit failed.
```

**Comment by Shougo** [comment]
I cannot reproduce the problem.
Please re-install deoplete-go.

**Comment by mfzl** [comment]
I tried removing and then installing, still gives me the error:

```
/usr/bin/git submodule update --init
error: no such remote ref 1356d8cd3593b1abb2f291e2cabf5430dfa108ed
Fetched in submodule path 'rplugin/python3/deoplete/clang', but it did not contain 1356d8cd3593b1abb2f291e2cabf5430dfa108ed. Direct fetching of that commit failed.
make: *** [Makefile:29: build/] Error 1
```

**Comment by zchee** [comment]
@faxal Hi, sorry for late reply :(

I'm tested, not reproduced, Hmm...
but I fixed libclang-python3 repository; that's a separate branch by the libclang versions.
maybe related...? I don't know.

Anyway, `1356d8cd3593b1abb2f291e2cabf5430dfa108ed` is `HEAD~`, actually this https://github.com/zchee/libclang-python3/commit/1356d8cd3593b1abb2f291e2cabf5430dfa108ed commit.
So, if you have libclang shared library, Please remove submodule and `git clone` into `rplugin/python3/deoplete/clang` path by the corresponding branch that libclang version of your having.

and I will fix `deoplete-go` submodule to `3.6` branch because some standard Linux distro does not provided clang-3.9(latest and master) package such as Ubuntu family expects `xenial`.
I think solve your problem after that's work.

**or**, if you do not want `C` completion, nothing to do.
`g:deoplete#sources#go#cgo` default is disable. will not `import` clang module because https://github.com/zchee/deoplete-go/blob/master/rplugin/python3/deoplete/sources/deoplete_go.py#L53-L56

**Comment by mfzl** [comment]
hey, I can install fine now with after I fetched the latest updated. Not sure which one though.
